src/openforms/contrib/haal_centraal/admin.py

```diff
@@ -3,15 +3,25 @@
 
 from solo.admin import SingletonModelAdmin
 
-from .models import HaalCentraalConfig
+from .models import BRPPersonenRequestOptions, HaalCentraalConfig
 
 
 @admin.register(HaalCentraalConfig)
 class HaalCentraalConfigAdmin(SingletonModelAdmin):
     fieldsets = (
         (
             _("BRP Personen Bevragen"),
-            {"fields": ("brp_personen_service", "brp_personen_version")},
+            {
+                "fields": (
+                    "brp_personen_service",
+                    "brp_personen_version",
+                    "default_brp_personen_purpose_limitation_header_value",
+                    "default_brp_personen_processing_header_value",
+                )
+            },
         ),
     )
     autocomplete_fields = ("brp_personen_service",)
+
+
+admin.site.register(BRPPersonenRequestOptions)
```

Comment:
please spend some effort on nice admin integration, `admin.site.register(model)` is almost never good enough.

In particular, ensure we have proper:

* `list_display`
* `list_select_related`
* `list_filter`
* `search_fields`
* `raw_id_fields` (the form FK will render a dropdown with an excessive amount of options)